tmlib/workflow/illuminati/api.py

```diff
@@ -396,8 +404,11 @@ def _create_maxzoom_level_tiles(self, batch, assume_clean_state):
                 image_store = dict()
                 image = file.get()
                 if batch['illumcorr']:
-                    logger.debug('correct image')
-                    image = image.correct(stats)
+                    if not layer.channel.name in non_illumcorr_channels:
```

Comment:
`if layer.channel.name not in non_illumcorr_channels:` is a bit more readable.

However, the condition still contains a double negation ("not in non_...") so it would be best to swap the two branches of the `if` and avoid the double negation altogether::

    if layer.channel.name in non_illumcorr_channels:
        logger.info("Not applying ...")
    else:
        logger.debug("Correcting ...")
        image = ...